NAMESPACE

```diff
@@ -1,2 +1,11 @@
 # Generated by roxygen2: do not edit by hand
 
+export(all_candidates)
```

Comment:
generally we want function names to start with a verb. So `read_all_candidates()` might be better, since the effect will be to return a `tibble`, which is similar to the behavior of `read_csv()`.